ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -36,15 +47,22 @@
     private ApiProviderWizard wizard = new ApiProviderWizard();
 
     private ReviewApiProviderActions reviewApiProviderActions = new ReviewApiProviderActions();
-    private final OperationsList operationsList = new OperationsList(By.className("operations-list"));
+    private ApiProviderToolbar toolbar = new ApiProviderToolbar();
 
-    @When("^select operation (.+)$")
+    @When("^select operation flow (.+)$")
     public void selectOperation(String operationName) {
-        OperationsList operationsList = new OperationsList(By.className("operations-list"));
+        OperationsList operationsList = new OperationsList(By.cssSelector("syndesis-integration-api-provider-operations-list pfng-list"));
         operationsList.validate();
         operationsList.invokeActionOnItem(operationName, ListAction.CLICK);
     }
 
+    @Then("check ([\\w ]+) operation not present in operation list")
```

Comment:
do we need more verbs here to make it look like a sentence or no?

Reply:
sounds fine to me, but I can add an "is" to make it clearer